### PR TITLE
[REF] EU emission factors 2022

### PR DESCRIPTION
**Pull request by mathilde-daugy**
Issue
Update US emission factors with EU-ETS 2022 values. The process is the same as last year, see [wiki](https://github.com/electricitymaps/electricitymaps-contrib/wiki/EU-emission-factors) for documentation

Computation notebook: [EU_emission_factors.ipynb](https://colab.research.google.com/drive/1buA-yHQCF711F53IX7_GAxYMJ9p4qJoT?usp=sharing)

Description
Updated yaml files for all EU zones that have REF

Preview
NA

Double check
 I have tested my parser changes locally with poetry run test_parser "zone_key"
 I have run pnpx prettier --write . and poetry run format to format my changes.

### CODE REVIEW

File: config/zones/ES.yaml [hunk 1]
```diff
@@ -54,13 +54,19 @@ emissionFactors:
...
     coal:
-      datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
-      value: 971.2573613
+      - datetime: '2021-01-01'
+        source: EU-ETS, ENTSO-E 2021
+        value: 971.2573613
+      - datetime: '2022-01-01'
+        source: EU-ETS, ENTSO-E 2022
+        value: 1156.28
```
Comment (mathilde-daugy) [inline config/zones/ES.yaml:62]: Carbon intensity factor is computed using data from the 2 operating coal PPs

File: config/zones/HU.yaml [hunk 2]
```diff
@@ -49,13 +49,19 @@ emissionFactors:
...
     coal:
-      datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
-      value: 1144.916957
+      - datetime: '2021-01-01'
+        source: EU-ETS, ENTSO-E 2021
+        value: 1144.916957
+      - datetime: '2022-01-01'
+        source: EU-ETS, ENTSO-E 2022
+        value: 1211.37
```
Comment (mathilde-daugy) [inline config/zones/HU.yaml:57]: Only 1 coal PP which seems to burn lignite hence the high ED

File: config/zones/IE.yaml [hunk 3]
```diff
@@ -48,13 +48,19 @@ emissionFactors:
...
     coal:
-      datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
-      value: 758.4409086
+      - datetime: '2021-01-01'
+        source: EU-ETS, ENTSO-E 2021
+        value: 758.4409086
+      - datetime: '2022-01-01'
+        source: EU-ETS, ENTSO-E 2022
+        value: 1082.54
```
Comment (mathilde-daugy) [inline config/zones/IE.yaml:56]: We used data from 2 pps to compute this factor but the increase is important so we might want to investigate more

File: config/zones/IE.yaml [hunk 4]
```diff
@@ -48,13 +48,19 @@ emissionFactors:
...
     gas:
-      datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
-      value: 396.8903896
+      - datetime: '2021-01-01'
+        source: EU-ETS, ENTSO-E 2021
+        value: 396.8903896
+      - datetime: '2022-01-01'
+        source: EU-ETS, ENTSO-E 2022
+        value: 624.48
```
Comment (mathilde-daugy) [inline config/zones/IE.yaml:63]: a bit high, we can consider revising the upper limit for gas EF (right now at 1700 g/kWh)

**Comment by VIKTORVAV99** [comment]
Just to double check, this only adds the new emission factors right? It does not update any current ones if any data or power plant mapping has changed?

**Review by VIKTORVAV99 — APPROVED** [review]
Didn't see any major weirdness besides those that you pointed out so this looks good to me! 🎉

**Review by pierresegonne — APPROVED** [review]

**Review by florianscheidl — APPROVED** [review]
All looks fine to me. Changes are not as significant as for the US and I think you commented on most of the large changes.

**Comment by cgicgi** [comment]
Knowing that I am a bit late to the party, but would like to ask @mathilde-daugy if you have been able to check out the work that w-flo has done in https://github.com/w-flo/eu-emission-factors based on the discussion from https://github.com/electricitymaps/electricitymaps-contrib/issues/5720

If I see it correctly, the inbalance of DE gas emission factors has increased with this update (EF 472 in 2021, 613 in 2022) and is hardly plausible. w-flo's work could reduce this potential inaccuracy.

**Pull request merged**